common/utils/budget-default.test.js

```diff
@@ -464,17 +459,28 @@ describe('budget getDefault methods', () => {
             total: { total: 0, federal: 0, medicaid: 0, state: 0 }
           }
         },
-        mmisByFFP: {
-          '90-10': {
+        ddi: {
```

Comment:
this should be returning this format, because we want to store the totals by split type
```
ddi: {
        '90-10': {
            type: Map,
            of: fedStateSplitByCost
        },
        '75-25': {
            type: Map,
            of: fedStateSplitByCost
        },
        '50-50': {
            type: Map,
            of: fedStateSplitByCost
        },
        combined: {
            type: Map,
            of: fedStateSplitByCost
        }
    },
```

Reply:
We need these values to fill out these tables:
https://www.figma.com/file/hJpKHKU6fz5J0Z7fisSwa2/eAPD-MMIS-2022?node-id=3263%3A18525&t=erGP3jnQ54R2O6K2-1
https://www.figma.com/file/hJpKHKU6fz5J0Z7fisSwa2/eAPD-MMIS-2022?node-id=4296%3A20589&t=uPYr3CVJJMFyVHph-1

Reply:
Ah, 🤦‍♀️  that's a misunderstanding on my part- I did start out with the splits in the model, but thought we didn't need it. The figma links help! I was focused on a different figma page where the tables didn't need the data further broken down. Fixing it now!